content/general/concepts/environment-variables/environment-variables.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Environment Variables'
+Description: 'Environment variables consist of a name/value pair and are set outside of applications, but accessible to them from the environment in which they are run.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Computer Science'
+Tags:
+  - 'Bash/Shell'
+  - 'Command Line'
+  - 'Linux'
+  - 'Unix'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/computer-science'
+---
+
+**Environment variables** consist of a name/value pair and are set outside of applications, but accessible to them from the environment in which they are run.
+
+Environment variables are often set via the [terminal](https://www.codecademy.com/resources/docs/general/terminal) or in a script, though how they are managed and retrieved varies depending on the operating system or environment. For example, they can be used to store system configuration data or private keys for applications.
```

Comment:
```suggestion
Environment variables are often set via the [terminal](https://www.codecademy.com/resources/docs/general/terminal) or in a script. They are managed and retrieved depending on the operating system or environment. For example, they can be used to store system configuration data or private keys for applications.
```

Reply:
@yangc95 thanks for this suggestion! I changed the code and slightly reworded the sentence in a way I thought was clearer, but I can change to the above suggestion exactly if you think that is better.